src/components/DifficultyRating.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <div class="flex items-center gap-1">
+    <span v-for="n in 3" :key="n" class="text-yellow-400 text-xl">
+      <span v-if="n <= difficulty">★</span><span v-else>☆</span>
+    </span>
+    <span class="ml-2 text-sm text-gray-500">(Dificultad: {{ difficulty }})</span>
```

Comment:
```suggestion
    <span class="dificultad">(Dificultad: {{ difficulty }})</span>
```
pongamos nombres que representen la clase o directamente o poner class="" hasta que necesitemos asignarle un nombre

Reply:
directamente no poner class

Reply:
```suggestion
    <span>(Dificultad: {{ difficulty }})</span>
```